TYPO3.Fluid/Classes/TYPO3/Fluid/ViewHelpers/RenderViewHelper.php

```diff
@@ -101,9 +137,34 @@ public function render($section = null, $partial = null, $arguments = array(), $
     }
 
     /**
+     * Loads the "argument" child nodes.
+     *
+     * @param $arguments
+     *
+     * @return array
+     */
+    protected function loadArgumentChildrenIntoArguments($arguments)
+    {
+        foreach ($this->childNodes as $childNode) {
+            if ($childNode instanceof ViewHelperNode
+                && $childNode->getViewHelperClassName() === \TYPO3\Fluid\ViewHelpers\ArgumentViewHelper::class
```

Comment:
If the Namespace is imported this could be put into one line:

```
if ($childNode instanceof ViewHelperNode $childNode->getViewHelperClassName() === ArgumentViewHelper::class) {
```